scripts/movement/release-name

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -eou pipefail
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+# Get the cargo workspaces version by matching the line version = "x.x.x" directly to avoid add dependencies
+CARGO_WORKSPACE_VERSION=$(awk -F\" '/version =/ {print $2}' Cargo.toml | head -n 1)
+# replace the . with - to avoid docker tag issues
```

Comment:
What are the issues? I've seen plenty of containers with dots in tag names on Docker repositories.

Reply:
Yeah, I'm even tagging current containers with dots. I was initially thinking of something down the line in infra. But, it's actually not relevant.